### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by SergeiSavenko**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+info:
+  app:
+    name: Lexigram service
+    description: Proxy to Lexigram API with some additional methods
+    version: 1.0.0
+    docs: http://docs.ymdlexigram.apiary.io
```
Comment (pedropregueiro) [inline src/main/resources/application.yml]: had a look at the current docs. nice work there!

**Comment by vincentp** [comment]
Can we have support for /search?

**Comment by SergeiSavenko** [comment]
Is it what you need?
http://zuul.dev.your.md/lexigram-service/search?q=sore%20throat

**Comment by vincentp** [comment]
Yes perfect! I didn't see it in the doc :) Can I use it now?

**Comment by SergeiSavenko** [comment]
Sure you can! It was a secret feature:)

**Pull request merged**